src/contexts/AnalyticsContext.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { nanoid } from 'nanoid'
+import { PostHogProvider, usePostHog } from 'posthog-react-native'
+import { ReactNode, useCallback, useEffect } from 'react'
+
+import { useAppDispatch, useAppSelector } from '~/hooks/redux'
+import { getWalletsMetadata } from '~/persistent-storage/wallets'
+import { analyticsIdGenerated } from '~/store/settingsSlice'
+
+const PUBLIC_POSTHOG_KEY = 'phc_CGYfA9jfoeMKXW629pQs1vwI1hxD3icXgnTDezOhaGz'
```

Comment:
This is a personal project of mine, it is not under the Alephium organization. We need to upgrade our account in order to have multiple projects.

Reply:
#175